app/services/get_system_metrics.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'sidekiq/api'
+
+class GetSystemMetrics
```

Comment:
I think `SystemMetrics` is a better name for this class than `GetSystemMetrics`.